test/module/shared_model/validators/validators_fixture.hpp

```diff
@@ -18,17 +18,28 @@
 #ifndef IROHA_VALIDATORS_FIXTURE_HPP
 #define IROHA_VALIDATORS_FIXTURE_HPP
 
+#include <google/protobuf/dynamic_message.h>
+#include <google/protobuf/reflection.h>
+#include <google/protobuf/stubs/casts.h>
 #include <gtest/gtest.h>
 #include <boost/range/adaptor/transformed.hpp>
```

Comment:
Same for ordering

Reply:
clang-format doesn't do anything about this.